### ISSUE TITLE
Is root-tag <RequestSecurityTokenResponseCollection> really necessary?

### ISSUE THREAD

**Issue by doeringp** [616b2f/IdentityServer4.WsFederation#4]
I opened this issue for discussion. Look at this [TODO comment](https://github.com/616b2f/IdentityServer4.WsFederation/blob/dd40e7a67095bf166f32c2f330a92c004056be60/src/IdentityServer4.WsFederation/ResponseProcessing/RequestSecurityTokenResponse.cs#L34) in code.

In my opinion wrapping the `<RequestSecurityTokenResponse>` in a `<RequestSecurityTokenReponseCollection>` isn't necessary.

The old sample [IdentityServer/IdentityServer4.WsFederation](https://github.com/IdentityServer/IdentityServer4.WsFederation) running on .NET Framework, does not do it either.

I've tested it on SharePoint. It did work without the collection:
![image](https://user-images.githubusercontent.com/20394732/99960725-85997f00-2d8d-11eb-9e00-cf3eb7ea9132.png)
![image](https://user-images.githubusercontent.com/20394732/99960735-88946f80-2d8d-11eb-9132-406327023eae.png)

**Comment by doeringp** [comment]
ADFS also does not wrap the RequestSecurityTokenResponse in a collection.

**Comment by 616b2f** [comment]
You are right we could omit it to be better compatible (more ADFS like). PR's are welcome :)
